src/hash.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+class Hashtable(object):
+    """Implement a hashtable."""
+
+    def __init__(self, size=11):
+        """Initialize table."""
+        self.size = size
+        self.table = [[] for x in range(self.size)]
+
+    def set(self, key, value):
+        """Store value at given key."""
+        if isinstance(key, str):
+            hashed_key = self._hash(key)
+            self.table[hashed_key].append((key, value))
```

Comment:
What happens if you input the same key again with a new value? This method wouldn't re-set that value. It'd just append the new key-value pair to the same bucket (due to a collision). Then you wouldn't be able to access your newly-set value, you'd only access the initial value.